server/src/test/java/org/candlepin/dto/api/v1/RoleDTOTranslatorTest.java

```diff
@@ -0,0 +1,136 @@
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.dto.api.v1;
+
+import org.candlepin.dto.AbstractTranslatorTest;
+import org.candlepin.dto.ModelTranslator;
+import org.candlepin.model.PermissionBlueprint;
+import org.candlepin.model.Role;
+import org.candlepin.model.User;
+
+import java.time.ZoneOffset;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * Test suite for the RoleDTOTranslator class.
+ */
+public class RoleDTOTranslatorTest extends
+    AbstractTranslatorTest<RoleDTO, Role, RoleDTOTranslator> {
+
+    protected RoleDTOTranslator translator = new RoleDTOTranslator();
+
+    protected PermissionBlueprintDTOTranslatorTest permDTOTranslatorTest =
+        new PermissionBlueprintDTOTranslatorTest();
+
+    protected UserDTOTranslatorTest userDTOTranslatorTest = new UserDTOTranslatorTest();
+
+    @Override
+    protected void initModelTranslator(ModelTranslator modelTranslator) {
+        this.userDTOTranslatorTest.initModelTranslator(modelTranslator);
+        this.permDTOTranslatorTest.initModelTranslator(modelTranslator);
+
+        modelTranslator.registerTranslator(this.translator, RoleDTO.class, Role.class);
+    }
+
+    @Override
+    protected RoleDTOTranslator initObjectTranslator() {
+        return this.translator;
+    }
+
+    @Override
+    protected RoleDTO initSourceObject() {
+        RoleDTO source = new RoleDTO();
+
+        List<UserDTO> users = new ArrayList<>();
+        List<PermissionBlueprintDTO> permissions = new ArrayList<>();
+
+        for (int i = 0; i < 5; ++i) {
+            UserDTO user = (UserDTO) this.userDTOTranslatorTest.initSourceObject();
+            user.setId("test-user-" + i);
+
+            PermissionBlueprintDTO permission = (PermissionBlueprintDTO)
+                this.permDTOTranslatorTest.initSourceObject();
+            permission.setId("test-perm-" + i);
+
+            users.add(user);
+            permissions.add(permission);
+        }
+
+        source.id("test-role-id")
+            .name("test-role-name")
+            .users(users)
+            .permissions(permissions)
+            .created(new Date().toInstant().atOffset(ZoneOffset.UTC))
+            .updated(new Date().toInstant().atOffset(ZoneOffset.UTC));
```

Comment:
No need to create a new Date object and then transform it into OffsetDateTime. You can use `OffsetDateTime.now()` directly

Reply:
Done.